x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/rules_notification_alert_type.ts

```diff
@@ -53,22 +53,22 @@ export const rulesNotificationAlertType = ({
       from: fromInMs,
       to: toInMs,
       index: ruleParams.outputIndex,
-      ruleId: ruleParams.ruleId!,
+      ruleId: ruleParams.ruleId,
       callCluster: services.callCluster,
     });
 
     const resultsLink = getNotificationResultsLink({
       from: fromInMs,
       to: toInMs,
       id: ruleAlertSavedObject.id,
-      kibanaSiemAppUrl: ruleAlertParams.meta?.kibanaSiemAppUrl as string,
```

Comment:
`RuleAlertParams` were all camelcase, previously (`falsePositives`, `maxSignals`, etc.)

Reply:
Agreed, but meta is something coming from the ui and the backend doesn't have much of an opinion about it other than to try to keep it snake case.

I am fine for now keeping this one inconsistency but over REST having snake case consistency. The others are transformed to snake case at least exiting. It might make things easier to use more snake case in spots like alerting params?